src/features/colinks/CoLinksContext.tsx

```diff
@@ -99,13 +101,13 @@ const CoLinksProvider: React.FC<CoLinksProviderProps> = ({ children }) => {
   //   return <LoadingIndicator />;
   // }
 
-  if (data === undefined) {
+  if (isLoading) {
     return <LoadingModal visible={true} />;
   }
 
-  if (!data) {
-    return <Text>Loading...</Text>;
-  }
+  // if (!data) {
+  //   return <Text>Loading...</Text>;
```

Comment:
i think we can remove this... not sure if theres an error case here we are not handling tho